src/components/search.js

```diff
@@ -99,18 +92,19 @@ class Search extends Emitter {
                      * @event Chat#$"."error"."history
                      */
                     this.chatEngine.throwError(this, 'trigger', 'search', new Error('There was a problem searching history. Make sure your request parameters are valid and history is enabled for this PubNub key.'), status);
-
                 } else {
+                    const startDate = response.startTimeToken;
+                    this.referenceDate = response.startTimeToken;
```

Comment:
I used separate variable to store `response.startTimeToken`, because value has been used in at least two places. It's usage increase length of code line which I tried to keep smaller so there will be no need to scroll horizontally to see the code.
If it is not so important, I can remove `startDate` variable and pass longer version to used code.

Reply:
@ianjennings  There is at least two places where it used in this piece of code and it is pretty long. I tried to keep line of code shorter, so there will be no need to do horizontal scroll to see it.
If it is bad, then I can replace with longer versions.

Reply:
It looks like ```starDate === thisreferenceDate```, so use one?